DependencyInjection/EzSystemsTweetFieldTypeExtension.php

```diff
@@ -18,8 +18,16 @@ class EzSystemsTweetFieldTypeExtension extends Extension implements PrependExten
 {
     public function prepend(ContainerBuilder $container)
     {
-        $config = Yaml::parse(file_get_contents(__DIR__ . '/../Resources/config/ez_field_templates.yml'));
-        $container->prependExtensionConfig('ezpublish', $config);
+        $configFiles = [
+            'ez_platformui' => 'config/yui.yml',
+            'ezpublish' => 'config/ez_field_templates.yml'
+        ];
+
+        foreach ($configFiles as $extensionName => $configFileName) {
+            $configFile = __DIR__ . '/../Resources/' . $configFileName;
```

Comment:
nitpick: you can move this part into the `$configFiles` values to get rid of this line, seems to not reach line length limits by good margin.